### PR TITLE
support custom object

### PR DESCRIPTION
**Pull request by localvar**

### CODE REVIEW

**Review by zhao-kun — COMMENTED** [review]
Need unit test to for the coverage

File: emctl/cmd/client/command/get/getter.go [hunk 1]
```diff
@@ -59,11 +58,11 @@ func WrapGetterByMeshObject(object meta.MeshObject,
 		return &observabilityTracingsGetter{object: object.(*resource.ObservabilityTracings), baseGetter: base}
 	case resource.KindIngress:
 		return &ingressGetter{object: object.(*resource.Ingress), baseGetter: base}
+	case resource.KindCustomObjectKind:
+		return &customObjectKindGetter{object: object.(*resource.CustomObjectKind), baseGetter: base}
 	default:
-		common.ExitWithErrorf("BUG: unsupported kind: %s", object.Kind())
+		return &customObjectGetter{object: object.(*resource.CustomObject), baseGetter: base}
```
[the remark this answers is not in the document]
Comment (xxx7xxxx) [reply emctl/cmd/client/command/get/getter.go:64]: I have the same concern too before. The mesh controller(server-side) will report the error. It's like CRD in k8s.

[the remark this answers is not in the document]
Comment (localvar) [reply emctl/cmd/client/command/get/getter.go:64]: yes, the server-side will report an error if kind does not exist.
validate `kind` requires sending an additional request to the server-side, which is not necessary,

**Comment by localvar** [comment]
> Need unit test for the coverage

test cases added.

**Review by zhao-kun — APPROVED** [review]
LGTM

**Review by xxx7xxxx — APPROVED** [review]

**Pull request merged**